components/dashboard/script.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+
+const APP_NAME = "Gitpod";
+
+const vars = {
+    REACT_APP_HOSTNAME : APP_NAME,
+    REACT_APP_DOCS_URL : "https://www.gitpod.io/docs/",
+    REACT_APP_SOCIAL_URL : "https://community.gitpod.io/",
+    REACT_APP_FAVICON_192 : "./favicon192.png",
+    REACT_APP_FAVICON_256 : "./favicon256.png",
+    REACT_APP_FAVICON_512 : "./favicon512.png",
+    REACT_APP_ICON_1 : "./icons/gitpod.svg",
+    REACT_APP_ICON_2 : "../icons/gitpod.svg",
+    REACT_APP_ICON_3 : "./images/gitpod.svg"
```

Comment:
Is 3 the same image as 1,2 but in difficult location?

Reply:
No its a different image with same name

Reply:
In that case change the variable name

Reply:
Done